client/src/style/components.css

```diff
@@ -0,0 +1,27 @@
+header{
```

Comment:
this file should live under header.css - and only imported in Header.js component.